src/hal/src/image.rs

```diff
@@ -368,8 +370,10 @@ impl Usage {
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum WrapMode {
-    /// Tile the texture. That is, sample the coordinate modulo `1.0`. This is
-    /// the default.
+    /// Tile the texture, that is, sample the coordinate modulo `1.0`, so 
+    /// addressing the texture beyond an edge will "wrap" back from the
+    /// other edge.
+    /// This is the default.
```

Comment:
I don't think there is a default any more

Reply:
Is there a concrete default then?

Reply:
Not really. Neither `WrapMode` or `SamplerInfo` (which the `WrapMode` is a part of) implement `Default`, and the only constructor of the former gets `WrapMode` as an argument.